index.html

```diff
@@ -332,124 +279,31 @@
 			}
 	});
 
-				addVectorLayer(
```

Comment:
These layers were being loaded but never referenced so I cut them.  In theory this should improve performance a little.  I haven't actually noticed a difference, but the pruning will make future maintenance easier.